test/test_bindings.py

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env python
+
+import unittest
+
+import numpy as np
+
+from sdf_tools import utils_2d
+
+
+class TestSDFTools(unittest.TestCase):
+
+    def test_sdf_tools(self):
+        res = 0.05
+        x_width = 20
+        y_height = 40
+        grid_world = np.zeros([y_height, x_width], dtype=np.uint8)
+        grid_world[1, 3] = 1
+        center_x = 0
+        center_y = 0
+        sdf_origin = [center_x - x_width / 2, center_y - y_height / 2]
+
+        sdf, sdf_gradient = utils_2d.compute_sdf_and_gradient(grid_world, res, sdf_origin)
+
+        self.assertAlmostEqual(sdf[1, 3], -res)
```

Comment:
:+1: 

These look good. Real tests for the correctness of the SDF creation should be in C++.